src/app/interfaces/storage.interface.ts

```diff
@@ -17,6 +17,7 @@ export interface VDev {
   stats: VDevStats;
   device?: string;
   disk?: string;
+  name?: string;
```

Comment:
I would prefer to keep backend interfaces clean and add extra fields to a new interface.

Reply:
When VDev has type VDevType.Mirror then name property is present in the API response